src/front/wgsl/mod.rs

```diff
@@ -12,7 +12,7 @@ use crate::{
     proc::{
         ensure_block_returns, Alignment, Layouter, ResolveContext, ResolveError, TypeResolution,
     },
-    FastHashMap,
+    FastHashMap, ScalarKind,
```

Comment:
Let's not include IR stuff at module scope here, except for the container wrappers (hashmap, arena, handle). This would be inconsistent with the rest of the codebase.